task_1/6.js

```diff
@@ -11,10 +11,12 @@ someFn(‘javaScript’) // JavascripT
 
 const someFn = (someString) => {
     const result = someString.toLowerCase();
+    const firstLetter = result[0].toUpperCase();
+    const lastLetter = result[result.length - 1].toUpperCase();
```

Comment:
this operation should happen only if your statement is truthful. No matter to do tis in case when someString.length is odd